arangod/RestServer/Metrics.h

```diff
@@ -515,6 +523,13 @@ template<typename Scale> class Histogram : public Metric {
       result += "{" + ls + "}";
     }
     result += " " + std::to_string(sum) + "\n";
+    if (alternativeName.empty()) {  // This is version 2 of the API
```

Comment:
Well, this is a temporary hack for 3.8, because the v1 is supposed to be removed with 3.9, I could add a separate argument but chose to do it this way since it is a temporary solution. Let's discuss in a call what I should do now.

Reply:
I can live with that if it is indeed temporary.